apps/documentation/src/styles.scss

```diff
@@ -1,4 +1,2 @@
-/* You can add global styles to this file, and also import other style files */
-:root {
-  --ng-doc-app-max-width: auto;
-}
```

Comment:
I agree that `auto` is not ideal, but the default max-width is a bit too small. How about `1600px` instead? That will remove the horizontal scrollbars in the Summary Range Selector demo without stretching too far on ultrawide screens.